gui/slick/interfaces/default/home.mako

```diff
@@ -222,7 +224,7 @@ $(document).ready(function(){
             4: { sorter: 'quality' },
             5: { sorter: 'eps' },
             % if sickbeard.FILTER_ROW:
-            6: { filter : 'parsed' }
+                7: { filter : 'parsed' }
             % endif
         },
```

Comment:
those are used by the column selector though, and reording the columns no? I would assume this could be the cause of the issue @fernandog had with testing this, because he has different columsn selected and/or a different order of the columns.